company/values.md

```diff
@@ -1,15 +1,34 @@
 # Sourcegraph values
 
-Our values are:
+Our values reflect who we are and how we work.
 
-## People
+## People first
 
-Together we are advancing technology for the good of people all around the world. We will attract, hire and retain the best teammates in the world and treat everyone in a first-class manner.
+[Our amazing team](teammate_qualities.md) is what makes Sourcegraph special. Together we are advancing technology for the good of people all around the world and we treat everyone in a first-class manner.
 
-## Journey
+## Customer driven progress
```

Comment:
This paragraph reminded me of "Validated learning" concept from "Lean startup"  book. If that was the reference for it maybe make it more explicit and actionable?
Something like:
```
We have a vision and to achieve it we are committed to learn from our customers, validate our assumption quickly via small and intentional iterations
```

Reply:
What I like about this quote ^ is that it applies not just to the vision for the whole product but also for features and product areas, that it gives room for someone on the team to have an idea, a vision for feature, that may not be derived from customer feedback, but that the teammate makes a strong case for why they believe it would be valuable to customers. These are partially based on assumptions, _but we commit to learn from our customers and validate those assumptions quickly via small and intentional iterations_.